<launch>

  <!-- fetch or freight? -->
  <arg name="name" />

  <!-- set topics -->
  <arg name="map_topic" default="map" />
  <arg name="cmd_vel_topic" default="cmd_vel" />
  <arg name="odom_topic" default="odom" />

  <!-- planners, costmaps, etc -->
  <node pkg="move_base" type="move_base" name="move_base" respawn="true" output="screen">
    <param name="base_global_planner" value="deviation_global_planner/DeviationGlobalPlanner" />
    <param name="base_local_planner" value="deviation_local_planner/DeviationLocalPlanner" />
    <param name="recovery_behavior_enabled" value="false" />
    <param name="controller_frequency" value="25.0" />

    <remap from="cmd_vel" to="$(arg cmd_vel_topic)"/>
    <remap from="odom" to="$(arg odom_topic)"/>
    <remap from="map" to="$(arg map_topic)" />
  </node>

</launch>
